sklearn/tests/test_base.py

```diff
@@ -436,6 +436,7 @@ def __init__(self, attribute_pickled=5):
 
     def __getstate__(self):
```

Comment:
Remind me, @HolgerPeters, why this method does not call `super(...).__getstate__`...?